life_hc/MySQL/General/fn_insertVehicle.sqf

```diff
@@ -13,13 +13,14 @@ params [
     "_side",
     "_type",
     "_className",
+    "_shop",
```

Comment:
It's not expecting anything, if you passed any other type, it wouldn't raise an exception. For the sake of clarity (and the fact I'll be using its return at some point to avoid the silly checks). We are more than capable of using it fluently, the general end user *may* fall into a trap when no exception is given.